kraken/node_actions/abstract_node_scenarios.py

```diff
@@ -18,9 +18,10 @@ def node_stop_scenario(self, instance_kill_count, node, timeout):
         pass
 
     # Node scenario to stop and then start the node
-    def node_stop_start_scenario(self, instance_kill_count, node, timeout):
+    def node_stop_start_scenario(self, instance_kill_count, node, timeout, duration):
         logging.info("Starting node_stop_start_scenario injection")
         self.node_stop_scenario(instance_kill_count, node, timeout)
+        logging.info("Waiting for %s seconds before starting the node" % (duration))
```

Comment:
I think you want some sort of time.sleep(duration) here right? You are just logging the duration but not actually sleeping

Reply:
Right! Done.